### PR TITLE
docs: add primary domain scope section to identity brokering guide

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: docs/docs/guides/authentication/identity-brokering.md [hunk 1]
```diff
@@ -87,6 +87,28 @@ Google Example:
 
 ![Configure identity provider](/img/console_org_identity_provider.gif)
 
+### 4.Send the primary domain scope on the authorization request
+The default policy which will be shown on the login is always the ZITADEL default configruation. As soon as the user has entered the loginname and ZITADEL can identitfy to which organization he belongs, we will trigger the settings of the organization.
```
Comment (mffap) [inline docs/docs/guides/authentication/identity-brokering.md:91]:
```suggestion
ZITADEL will show a set of identity providers by default. This configuration can be changed by users with the [manager role] (https://docs.zitadel.ch/docs/concepts/zitadel/objects/managers) `IAM_OWNER`.

An organization's login settings will be shown 

- as soon as the user has entered the loginname and ZITADEL can identitfy to which organization he belongs; or
- by sending a primary domain scope.
```

File: docs/docs/guides/authentication/identity-brokering.md [hunk 2]
```diff
@@ -87,6 +87,28 @@ Google Example:
...
+To get your own configuration you will have to send the [primary domain scope](https://docs.zitadel.ch/docs/apis/openidoauth/scopes#reserved-scopes) in your [authorization request](https://docs.zitadel.ch/docs/guides/authentication/login-users/#auth-request) .
```
Comment (mffap) [inline docs/docs/guides/authentication/identity-brokering.md:92]:
```suggestion
You have to send the [primary domain scope](https://docs.zitadel.ch/docs/apis/openidoauth/scopes#reserved-scopes) in your [authorization request](https://docs.zitadel.ch/docs/guides/authentication/login-users/#auth-request) ..
```

**Review by mffap — APPROVED** [review]
@fgerschwiler I made two adjustments to the text to provide some more clarity, as I would understand. Original works as well, but is a bit hard to understand due to the terms (eg, 'default policy')

**Review by fforootd — APPROVED** [review]

**Comment by fgerschwiler** [comment]
> 

Yes, you are right, i committed your changes

**Pull request merged**